mottak-arkiv-service/tests/routers/dto/test_Arkivuttrekk.py

```diff
@@ -24,6 +24,7 @@ def _arkivuttrekk_domain() -> Arkivuttrekk:
         arkiv_sluttdato=date.fromisoformat("1900-05-12"),
         storrelse=45620,
         avtalenummer="01/12345",
+        depotinstitusjon="Arkivverket",
```

Comment:
Så matcher det forventet verdi
```suggestion
        depotinstitusjon="arkivverket",
```